src/webrtc/call.ts

```diff
@@ -236,7 +244,17 @@ export class MatrixCall extends EventEmitter {
     // The party ID of the other side: undefined if we haven't chosen a partner
     // yet, null if we have but they didn't send a party ID.
     private opponentPartyId: string;
-    private inviteTimeout;
+    private inviteTimeout: NodeJS.Timeout;
```

Comment:
is this actually a NodeJS timeout or a web browser timeout?

Reply:
here it's a browser one since webrtc is only in the browser, but in general in the js-sdk it could be either - I thought this is the type I'd seen used elsewhere though?

Reply:
it mostly depends on how we're clearing it - the node timeouts have a different contract than the browser ones, which can lead to runtime errors.

Reply:
have gone with the 'commenting it' option